cluster/charts/crossplane/templates/provider.yaml

```diff
@@ -0,0 +1,11 @@
+{{- range $.Values.provider.packages }}
+{{ if ne . "" }}
+apiVersion: pkg.crossplane.io/v1alpha1
+kind: Provider
+metadata:
+  name: {{ . | trim | replace "/" "-" | replace ":" "-" }}
+spec:
+  package: {{ . | trim }}
```

Comment:
I like that this is general now as opposed to us manually adding providers to a list that could be installed with Crossplane! 👍